### ISSUE TITLE
LicenseUrl is deprecated

### ISSUE THREAD

**Issue by lmolkova** [microsoft/ApplicationInsights-dotnet#1036]
When building in Release in VS or with buildRelease.cmd script, I get the error (warning as an error)

C:\Program Files\dotnet\sdk\2.1.500\Sdks\NuGet.Build.Tasks.Pack\buildCrossTargeting\NuGet.Build.Tasks.Pack.targets(202,5): error NU5125: The 'licenseUrl' element will be deprecated. Consider using the 'license' element instead. [C:\repo\ApplicationInsights-dotnet\src\
Microsoft.ApplicationInsights\Microsoft.ApplicationInsights.csproj]

The same happens on build server, however, build succeedes
https://dev.azure.com/mseng/AppInsights/_build/results?buildId=8086359

```
C:\Program Files\dotnet\sdk\2.1.502\Sdks\NuGet.Build.Tasks.Pack\buildCrossTargeting\NuGet.Build.Tasks.Pack.targets(202,5): Error NU5125: The 'licenseUrl' element will be deprecated. Consider using the 'license' element instead.
C:\Program Files\dotnet\sdk\2.1.502\Sdks\NuGet.Build.Tasks.Pack\buildCrossTargeting\NuGet.Build.Tasks.Pack.targets(202,5): Error NU5125: The 'licenseUrl' element will be deprecated. Consider using the 'license' element instead.
```

**Comment by MS-TimothyMothra** [comment]
I think this started happening with one of the Visual Studio updates.
The build server is always one or two updates behind. 
We'll have to fix this in the near future.

**Comment by SergeyKanzhelev** [comment]
closed via #1057
